### ISSUE TITLE
MagicaVoxel import only imports one section out of multiple

### ISSUE THREAD

**Issue by Metadorius** [guillaumechereau/goxel#208]
What the title says. When I try to import the VOX file from MagicaVoxel, only one of the multiple sections is imported.

**Comment by poVoq** [comment]
I am also trying to work with .vox files from the Veloren project and apparently only one part of the various models in the scene graph is imported. Here are some example files to test it: https://drive.google.com/drive/folders/1jkn8v9I0fo1nJlM29YAI_wWb_V-w456q

**Comment by guillaumechereau** [comment]
Right.  The current magica voxel import/export code is based on the old magica voxel format that didn't support layers.  I will see if I can get a reference to the new format and update the code.

**Comment by Metadorius** [comment]
> Right.  The current magica voxel import/export code is based on the old magica voxel format that didn't support layers.  I will see if I can get a reference to the new format and update the code.

One of my mates did a hardfork of Goxel for our own purposes (we lack a decent voxel editor for the games our community is modding) and implemented a partial support here: https://gitlab.com/TheChosenEvilOne/goxel/
Maybe you'd find that useful.

**Comment by guillaumechereau** [comment]
I just checked the new magica voxel format.  It's a bit complicated because they have concept of both layers and shapes.  I think I will first try to only support loading each magica voxel shape into a different goxel layer and ignore the magica voxel layers.

Also I realize material sync between goxel and magica voxel is never going to be good unless I implement palette based material like magica voxel does...
